### ISSUE TITLE
Continuity Service copy and paste not working. 

### ISSUE THREAD

**Issue by vink2212** [Bananz0/GalaxyBookEnabler#12]
Sorry for opening up another issue, seems like you've gotten most of the features working. The biggest feature that still does not work for me is the cross-platform copy and paste. No idea how i am supposed to get that to work, i remember it used to work for an older version of the script/ Samsung apps a few months ago but hasn't worked for me since.

**Comment by Bananz0** [comment]
Unfortunately seems like Samsung needs you to have multicontrol running for that, and even with that its so unreliable cause it sends the clipboard data over wifi and you need to be on the same wifi router, and not one with the laptop connected via ethernet and the phone to wifi or two wifi networks with the same name such as eduroam. I find it easier to use the Windows Phone Link for this that does this over bluetooth which is much more reliable and almost instant, only downside is you can only do it to one device.  :(

**Comment by vink2212** [comment]
Yeah, so I have actually met all the requirements you just listed since for some reason Multi-control does not work without my laptop and phone being on the same network. 
Every time I start my laptop up I get this notification from continuity service that 'To use copy and paste you must let continuity service run in the background ' 
When I click this notification it takes me to the app settings but I can't seem to find the place where I can configure background usage. 
Seems like the app is open in task manager but still nothing, multi-control works well with using the mouse and keyboard but the main reason I wanted to use this was the copy and paste feature.

**Comment by Bananz0** [comment]
I think the option to enable it run in the background is in the app settings, there should be an option like this.
![image](https://github.com/Bananz0/GalaxyBookEnabler/assets/31777541/d24d0ab7-ba12-46cf-8f5e-62336c7842cc)

**Comment by vink2212** [comment]
Lol for some reason I do not have that option lol. Let me update windows and see what is up, what version of windows 11 are you on ?

**Comment by Bananz0** [comment]
That's weird, I'm on 23H2 22631

**Comment by vink2212** [comment]
Yeah, I'm on 22H2, seems I have quite a bit of updating ahead of me. I will update and reset my PC and get back to you on this.
Does copy and paste work for you (after connecting both wifi and Bluetooth)?

**Comment by Bananz0** [comment]
Yes it does, but intermittently so i just switched to using the phone link indefinitely lol but it should work and no problem!

**Comment by vink2212** [comment]
Ok yes you were right, I was able to do some reg edit stuff to show me those settings (seems it's an issue with Windows having a bad install) and copy and paste work. Only seemed to work when I copied from laptop to phone. But your method of using phone link works much better, didn't know we could have multi-control also working in the background so this ends up being a perfect way to run the application. 
I'm gonna close this issue now, thanks a lot for the help @Bananz0

**Comment by AttackBaxter** [comment]
I know I had everything working EXTREMELY well with my old MSI gf65 thin running win10. While I did not use this program to install all the Samsung components needed to get things working (I wish I had found this before I spend probably 7 hours trying to figure every thing out back then), I instead installed all the exact versions of drivers and software needed to get SamSysSupSvc (Samsung System Support Services) to work allowing everything else to run(or at least that was my understanding of it). If its of any help to this project, even though Im assuming this has already been used or examined, here is what I used to get everything set up manually while it was working perfectly [https://xdaforums.com/t/how-to-samsung-multi-control-with-non-samsung-windows-pc.4640205/]
Also to note, before I found that I was also running galaxybook_mask (another github project) even after I set up everything else. I have no clue if this is of any help, but hopefully it is lol!
